pom.xml

```diff
@@ -40,37 +40,37 @@
 		<referenceapplicationVersion>2.12.0-SNAPSHOT</referenceapplicationVersion>
 		<providermanagementVersion>2.13.0</providermanagementVersion>
 		<uilibraryVersion>2.0.7</uilibraryVersion>
-		<calculationVersion>1.3.0-SNAPSHOT</calculationVersion>
+		<calculationVersion>1.3.0</calculationVersion>
```

Comment:
To revert last commit, you can say **git log** in your terminal , the copy the first commit id which is the last on github
then say  **git revert <last commit id>.** after push